### DISCUSSION TITLE
New Team Memeber

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#319] [Team Posts]
Dear all,

Today I would like to welcome Luca Riva to our team, who will join Adrien Crovato in looking after the French translations of our app.

@Lukesw74 Welcome! Thank you for helping.

Best,

Stefan.

**Comment by TTC70** [comment]
Welcome Luca.
...Lukesw74... Luke SkyWalker? 😁😁😁


Il Ven 20 Gen 2023, 07:42 Stefan Kebekus ***@***.***> ha
scritto:

**Comment by Lukesw74** [comment]
Hello

Hello all... happy to support and help...

Ciao
Luca


Il Ven 20 Gen 2023, 07:42 Stefan Kebekus ***@***.***> ha
scritto:

**Comment by Lukesw74** [comment]
Hello

... yes ... you got it.



Il Ven 20 Gen 2023, 08:17 TTC70 ***@***.***> ha scritto:

**Comment by TTC70** [comment]
hihihihih ... great.
I don't know if you need help, just in case you need, feel free to contact
me if you don't know how to use GitHub Desktop and linguist applications
(english or italian doesn't make any difference for me).
Ciao
Antonio

Il giorno ven 20 gen 2023 alle ore 08:56 Lukesw74 ***@***.***>
ha scritto:

**Comment by Lukesw74** [comment]
Ciao Antonio

thank for your support...

On the other end if you need support for the italian translation let me know... I am italian ot should be easy for me :-)

Ciao

**Comment by TTC70** [comment]
Well... I'll count on your help.
I'm Sicilian, then i might need some help 😎
See you on these screens. 😉

Il Dom 22 Gen 2023, 18:37 Lukesw74 ***@***.***> ha scritto:

**Comment by Lukesw74** [comment]
ahahah ...  I will check from time to time ...
